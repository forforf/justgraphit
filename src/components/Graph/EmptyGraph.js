import React from 'react';


const EmptyGraph = () =>
    <div>
      This is an EmptyGraph
    </div>;

export default EmptyGraph;
