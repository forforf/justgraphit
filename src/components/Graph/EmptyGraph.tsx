import React from 'react';

const EmptyGraph = (): JSX.Element => <div>This is an EmptyGraph</div>;

export default EmptyGraph;
