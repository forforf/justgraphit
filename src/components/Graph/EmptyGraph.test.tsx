import React from 'react';
import renderer from 'react-test-renderer';
import EmptyGraph from './EmptyGraph';

test('deep snapshot', () => {
  const component = renderer.create(<EmptyGraph />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
