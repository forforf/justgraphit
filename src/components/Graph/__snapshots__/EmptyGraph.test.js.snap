// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`deep snapshot 1`] = `
<div>
  This is an EmptyGraph
</div>
`;
